src/ToDoListView.test.js

```diff
@@ -35,11 +36,43 @@ describe('To Do List', () => {
     expect(axios.get).toHaveBeenCalledWith('/todos');
   });
 
+  it('should display main todo list', () => {
+    expect(getMainToDoList()).toBeInTheDocument();
+  });
+
   it('should display ToDos', () => {
     expect(screen.getByText('Item 1')).toBeInTheDocument();
     expect(screen.getByText('Item 2')).toBeInTheDocument();
   });
 
+  it('list elements should contain delete button', () => {
+    const list = getMainToDoList();
+    const { getAllByRole } = within(list);
+    const items = getAllByRole('listitem');
+    const listText = items.map((item) => item.textContent);
+    listText.forEach((value) => {
+      expect(value).toContain('Delete');
+    });
+  });
+
+  it('delete button should send delete request', () => {
+    axios.delete.mockImplementation(() => new Promise(jest.fn()));
+
+    userEvent.click(screen.getByLabelText('delete-button-1'));
```

Comment:
replace this with a lookup based on the list item text e.g. 
find list item by text 'Item 1', find the corresponding delete button